package.json

```diff
@@ -16,21 +16,18 @@
   "sideEffects": false,
   "scripts": {
     "build": "make build",
-    "build-docs": "build-storybook && npm run build-gatsby && npm run move-gatsby",
-    "build-storybook": "build-storybook",
+    "build-docs": "npm run build-gatsby && npm run move-gatsby",
     "build-gatsby": "cd ./www && \"$npm_execpath\" install && \"$npm_execpath\" run build",
     "commit": "commit",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --runInBand --coverage",
-    "deploy-storybook": "storybook-to-ghpages",
-    "deploy-storybook-ci": "storybook-to-ghpages --ci",
     "is-es5": "es-check es5 ./dist/*.js",
     "lint": "eslint --ext .js --ext .jsx .",
-    "move-gatsby": "mv ./www/public ./storybook-static/v2",
+    "move-gatsby": "mv ./www/public ./",
     "prepublishOnly": "npm run build",
     "semantic-release": "semantic-release",
     "snapshot": "jest --updateSnapshot",
-    "start": "start-storybook -p 6006",
+    "start": "cd ./www && \"$npm_execpath\" start",
```

Comment:
This will break for first time users of the repository since packages won't have been installed yet. We could just echo in terminal that /www is where the doc site lives and they must do npm install both in the top level directory and the www directory

Reply:
should we create an init command of some kind?

Reply:
Maybe `preinstall` would be better? https://stackoverflow.com/questions/48859022/override-npm-install-script-for-npm-project